dev/resources/views/widgets/images-missing-alt.blade.php

```diff
@@ -0,0 +1,39 @@
+<div class="card p-0 overflow-hidden h-full">
+    <div class="flex justify-between items-center pt-2 px-2">
+        <h2>
+            <div class="flex items-center">
+                <div class="h-6 w-6 mr-1 text-grey-80">
+                    @cp_svg('assets')
+                </div>
+                <span>{{ __('strings.widget_assets_title') }}</span>
+            </div>
+        </h2>
+    </div>
+    <div class="content p-2">
+        <p>{{ trans_choice('strings.widget_assets_explanation', $amount, ['amount' => $amount]) }}</p>
+    </div>
+
+    @if ($assets)
+        <table tabindex="0" class="data-table">
+            <tbody tabindex="0">
+    @endif
+    @forelse ($assets->slice(0, 5) as $asset)
```

Comment:
Maybe put this slice in the Controller so we only sent a max of 5 to the view.

Reply:
Good call!